src/components/types/Posts.ts

```diff
@@ -0,0 +1,14 @@
+import { User } from './User';
+import { Comments } from './comments';
+
+export interface Posts {
```

Comment:
Назви інтерфейсів, зазвичай, записуємо в однині
```suggestion
export interface Post {
```